restricted.ejs

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Restricted</title>
+  </head>
+  <body>
+    <p>This page is restricted. You can't see it unless you are logged on.</p>
+    <p>You have visited this page <%= pageCount %> times since logon.</p>
+  </body>
```

Comment:
Just an extra suggestion, it could be nice for the user to have a link to logout from here

Reply:
it is done👍